ot-node.js

```diff
@@ -380,37 +381,59 @@ class OTNode {
                 this.logger.warn(
                     `Unable to initialize Paranet with id ${paranetUAL} because of invalid UAL format`,
                 );
-            } else {
-                const { blockchain, contract, tokenId } = ualService.resolveUAL(paranetUAL);
-                if (!blockchainModuleManager.getImplementationNames().includes(blockchain)) {
+                continue;
+            }
+
+            const { blockchain, contract, tokenId } = ualService.resolveUAL(paranetUAL);
+            if (!blockchainModuleManager.getImplementationNames().includes(blockchain)) {
+                this.logger.warn(
+                    `Unable to initialize Paranet with id ${paranetUAL} because of unsupported blockchain implementation`,
+                );
+                continue;
+            }
+
+            const paranetId = paranetService.constructParanetId(blockchain, contract, tokenId);
+            // eslint-disable-next-line no-await-in-loop
+            const paranetExists = await blockchainModuleManager.paranetExists(
+                blockchain,
+                paranetId,
+            );
+            if (!paranetExists) {
+                this.logger.warn(
+                    `Unable to initialize Paranet with id ${paranetUAL} because it doesn't exist`,
+                );
+                continue;
+            }
+
+            // 0 - OPEN, 1 - CURATED
+            // eslint-disable-next-line no-await-in-loop
+            const nodesAccessPolicy = await blockchainModuleManager.getNodesAccessPolicy(
+                blockchain,
+                paranetId,
+            );
+            if (nodesAccessPolicy === 1) {
```

Comment:
We should use constant here instead of '1', it make code more readable and less likely we will make a mistake to put wrong literal